docs/VSCode Setup.md

```diff
@@ -0,0 +1,64 @@
+# Setting Up VSCode for Development
```

Comment:
Yet another piece of documentation that could've been helpful earlier. I've added this to help aid with vscode setup for the first time. This should've been created long ago but here we are